index.html

```diff
@@ -8,11 +8,12 @@
   <link rel="stylesheet" href="main.css">
 </head>
 <body>
+  <ul id="book_list"></ul>
```

Comment:
[Optional] It would be nice to also have the title `Awesome Books` at the top of the page, as seen in the wireframe
<img width="430" alt="image" src="https://user-images.githubusercontent.com/37116322/234072039-5111ae7a-7809-4f6d-8519-342582b87ea7.png">

Reply:
Thank you for your review and suggestion! We will add this for the next review.